applications/monitoring/values.yaml

```diff
@@ -77,20 +189,83 @@ cronjob:
     # -- taskmaker schedule
     taskmaker: "6-59/15 * * * *"
 
+# -- Gafaelfawr ingress for InfluxDBv2, only used if the service is enabled.
 ingress:
-  # -- ingress for Chronograf UI
-  chronograf:
-    # -- hostname for Chronograf UI
-    # @ default -- None, must be set by each individual instance
-    hostname: ""
+  enabled: false
+  influxdb2:
     # -- Additional annotations to add to the ingress
     annotations: {}
 
+# Telegraf for Prometheus monitoring.
+telegraf:
+  # -- enable telegraf at all?
+  enabled: true
+
+  # -- config for telegraf
+  # Use generated config instead of specifying anything here.
+  config:
+    processors: []
+    inputs: []
+    outputs: []
+
+  # -- pod resources for telegraf
+  resources:
+    limits:
+      memory: 512Mi
+      cpu: 900m
+
+  # -- arguments for telegraf invocation
+  args:
+  - "--config"
+  - "/etc/telegraf-generated/telegraf-generated.conf"
+
+  # -- rbac settings; we need the additional rules for Prometheus scraping.
+  rbac:
+    clusterWide: true
+
+  # -- environment for telegraf
+  env:
+  - name: INFLUX_TOKEN
+    valueFrom:
+      secretKeyRef:
+        key: telegraf-token
+        name: monitoring
+
+  # -- telegraf pod labels
+  # This annotation is required in an RSP so telegraf can get metrics from
+  # JupyterHub
+  podLabels:
+    hub.jupyter.org/network-access-hub: 'true'
+
+  # -- use telegraf service
+  service:
+    enabled: false
+
+  # -- template version
+  tplVersion: 2
+
+  # -- telegraf volume: generated configuration
+  volumes:
+  - name: telegraf-generated-config
+    configMap:
+      name: telegraf-generated-config
+
+  # -- telegraf volume mount: generated configuration
```

Comment:
Needs a `@default` comment to avoid dumping JSON into the README.

Reply:
telegraf/-ds removed